### PR TITLE
Change to github credits

### PR DESCRIPTION
**Pull request by Pottedplant530**
Gg i did a code

### CODE REVIEW

**Comment by nstringham** [comment]
This should probably also be updated
Screenshot_20201119-205557~2

File: README.md [hunk 1]
```diff
@@ -37,8 +37,12 @@ You can now run `npm start` to start it. You will be able to access the site on
...
-- **Animated Custom Weapons and Armors**: <a href="https://hypixel.net/threads/2138599/">FurfSky+</a> by Furf__
-- **Additional Custom Textures**: <a href="https://hypixel.net/threads/2147652/">Vanilla+</a> by TBlazeWarriorT
+- **Defualt Pack**: Combo of Furfsky+ and Vanilla+ by Leaphant
+- **Furfsky+ Pack**: <a href="https://hypixel.net/threads/2138599/">FurfSky+</a> by Furf__
+- **Vanilla+ Pack**: <a href="https://hypixel.net/threads/2147652/">Vanilla+</a> by TBlazeWarriorT
+- **FAR Block Pack**: <a href="https://hypixel.net/threads/far-block-hypixel-skyblock-resource-pack.2377411/">FAR Block</a> by Farkonong
+- **Hypixel Skyblock Pack**: <a href="https://hypixel.net/threads/hypixel-skyblock-pack-the-dungeon-update-i.2103515/">Hypixel Skyblock Pack</a> by Hypixel Packs Hq(doesn't work at the moment)
+- **Duckblock Pack**: <a href="https://hypixel.net/threads/outdated-go-to-new-link-x16-v4-0-beta-duckblock-a-skyblock-pack-purely-made-by-me.2794366/">Duckblock Pack</a> by Duckiscool69
 - **Default Textures**: <a href="https://www.minecraft.net/">Minecraft</a> by Mojang
 - **Player Heads**: <a href="https://hypixel.net/forums/skyblock.157/">SkyBlock</a> by Hypixel.
```
Comment (boblovespi) [inline README.md:47]:
```suggestion
- **Player Heads**: <a href="https://hypixel.net/forums/skyblock.157/">SkyBlock</a> by Hypixel
```

**Review by boblovespi — CHANGES_REQUESTED** [review]

**Comment by Pottedplant530** [comment]
alright @boblovespi I made the change, can you approve this

**Comment by Pottedplant530** [comment]
@boblovespi :|

**Review by metalcupcake5 — APPROVED** [review]

**Review by vyketype — CHANGES_REQUESTED** [review]
You put "Defualt Pack" in line 40. It should be "Default Pack".

**Review by nstringham — CHANGES_REQUESTED** [review]
Conflicts

**Pull request closed without merging**